app/src/app/account/credits/available-credits-banner.tsx

```diff
@@ -0,0 +1,32 @@
+import { redirect } from "next/navigation";
+
+import { InfoTooltip } from "~/components/info-tooltip";
+import { getProject } from "~/lib/server/projects";
+import { getUserFromSession } from "~/lib/session";
+
+export default async function AvailableCreditsBanner() {
+  const user = await getUserFromSession();
+
+  if (!user) {
+    return redirect("/login");
+  }
+
+  const horizonProject = await getProject(user?.accountId);
```

Comment:
You don't need to use optional chaining here since it is known that the user exists, otherwise the function would exit early 3 lines above.